### PR TITLE
[CI][VSTS] Move to ESRP for notarization.

### PR DESCRIPTION
**Pull request by mandel-macaque**
This PR contains just the required changes to add notarization via ESRP. Signing the pkg is done by the old python script (pay attention to the args). The artifacts are named the same and are in the same location, we only splinted the work.

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]

**Review by rolfbjarne — APPROVED** [review]

File: tools/devops/provision-brew-packages.csx [hunk 1]
```diff
@@ -4,8 +4,7 @@ BrewPackages (
     "automake",
     "libtool",
     "p7zip",
-    "python",
-    "libmagic",
```
Comment (spouliot) [inline tools/devops/provision-brew-packages.csx:8]: no more magic ?

**Review by spouliot — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]
👍👍 2x!

**Review by tj-devel709 — APPROVED** [review]

File: tools/devops/provision-brew-packages.csx (hunk 1 above)
Comment (mandel-macaque) [reply tools/devops/provision-brew-packages.csx:8]: I ran out of tricks!

**Comment by mandel-macaque** [comment]
This has a successful pkg: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4387372&view=logs&j=2efe22f3-7056-5903-a15e-77c42bdbe7c3&t=6b399b88-ec3b-5756-41e1-4097f4c09ccc

**Pull request merged**